README.es-ES.md

```
@@ -0,0 +1,174 @@
+    
+    MODEL_NODE --> SCREENSHOT
+    ENUM_NODE --> SCREENSHOT
+    SCREENSHOT --> DOWNLOAD
+    
+    DOWNLOAD -.-> PANEL_MGR
+    PANEL_MGR -.-> EXT_ENTRY
+```
+
+## 📦 Estructura del Proyecto
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)